cvat/apps/documentation/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently asked questions
+
+## How to update CVAT
+Before upgrading, please follow the oficcial docker [manual](https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes) and backup all CVAT volumes.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[maximum-line-length] Line must be at most 120 characters](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5666486)